pkgs/racket-doc/pkg/scribblings/pkg.scrbl

```diff
@@ -258,7 +258,7 @@ be accessed more efficiently when @nonterm{rev} refers to a branch or
 tag (even if it is written as a commit). In those cases, the content
 typically can be obtained without downloading irrelevant history.}
 
-For example, @filepath{http://bitbucket.org/game/tic-tac-toe#main}
```

Comment:
I think this is meant to be a dummy URL that shows what a Git package source is supposed to look like.